### ISSUE TITLE
Mandatory Font Verdana

### ISSUE THREAD

**Issue by alexseif** [mazedigital/Web-Ticker#36]
Great script, but the js forces a font type and style, which is really the CSS role and not for generic use.
https://github.com/mazedigital/Web-Ticker/blob/master/jquery.webticker.js#L236

**Comment by alessandrotesoro** [comment]
Was just about to post this. I don't see any need for this?

**Comment by jonmifsud** [comment]
Yes it was a complete oversight when one of our guys did the commit it slipped by! I'll try to get an update/fix up

**Comment by alessandrotesoro** [comment]
No worries, thanks for letting us know.
